src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/App.kt

```diff
@@ -176,6 +178,19 @@ fun App(
                 },
             )
         }
+
+        if (appState.project?.currentSampleFile != null) {
+            LaunchedEffect(Unit) {
+                appState.videoState.videoPath =
+                    FindVideoStrategy.SamePlaceOfReferenceAudio.find(
```

Comment:
Could you wrap this with a member function of `VideoState`?

Reply:
I gave an example in https://github.com/sdercolin/vlabeler/pull/17#discussion_r1033146249